include/SFML/Graphics/Color.hpp

```diff
@@ -62,6 +62,22 @@ class SFML_GRAPHICS_API Color
     Color(Uint8 red, Uint8 green, Uint8 blue, Uint8 alpha = 255);
 
     ////////////////////////////////////////////////////////////
+    /// \brief Construct the color from 32 bit unsigned integer
```

Comment:
It would be really nitpicking but shouldn't it be `32-bit` (with a dash)? Do we care?